frontend/src/apis/VendorsAPI.tsx

```diff
@@ -74,6 +74,17 @@ export const VENDORS_API = {
     });
   },
 
+  getVendorsNoPaginationBuyBackPolicy: async function (): Promise<APIVendor[]> {
+    return await API.request({
+      url: VENDORS_EXTENSION,
+      method: METHOD_GET,
+      params: {
+        no_pagination: true,
+        has_buyback_policy: true,
+      },
+    });
+  },
+
```

Comment:
Again, shouldn't really need another API here, just modify the one above to have a hasBuybackPolicy input